pages/index.css

```diff
@@ -2,3 +2,11 @@
 @import url(../vendor/fonts/fonts.css);
 @import url(../blocks/page/page.css);
 @import url(../blocks/page/__container/page__container.css);
+
+@import url(../blocks/research/research.css);
+@import url(../blocks/research/__image/research__image.css);
+@import url(../blocks/research/__link/research__link.css);
+@import url(../blocks/research/__paragrah/research__paragraph.css);
+@import url(../blocks/research/__texts/research__inner-wrapper.css);
```

Comment:
@Woolfs1423 Same here. Since you renamed your folder in BEM Nested, it has to change in the import statement as well. 
You've currently got your `research__inner-wrapper.css` file in `__inner-wrapper` directory, but here you try to access it within `__text` folder which doesn't exist anymore 😬